addon/components/property-group.hbs

```diff
@@ -20,40 +20,45 @@
       </div>
     {{/if}}
     {{!-- NOTE: Children can be property-groups as well as fields --}}
-    {{#each this.children as |child index|}}
-      {{#if (not (is-property-group child.displayType))}}
-        <div class="au-u-margin-bottom-small">
-          {{component (component-for-display-type child.displayType show=@show)
-                      field=child
-                      form=@form
-                      formStore=@formStore
-                      graphs=@graphs
-                      sourceNode=@sourceNode
-                      forceShowErrors=@forceShowErrors
-                      cacheConditionals=@cacheConditionals
-                      show=@show
-          }}
-          {{! template-lint-disable no-triple-curlies}}
-          <AuHelpText>{{{child.help}}}</AuHelpText>
-        </div>
-      {{else}}
-        <PropertyGroup
-                @level={{this.nextLevel}}
+    {{#each this.children as |field index|}}
+      {{#if field.displayType}}
+        {{#if (not (is-property-group field.displayType))}}
+          <div class="au-u-margin-bottom-small">
+            {{#let (component-for-display-type field.displayType show=@show) as |FormField|}}
```

Comment:
This won't work in < Ember 3.25 so in theory it's a breaking change. We can wrap it in an ensure-safe-component to make it work in older versions so we can release it as a 1.X release. In 2.0 we can then drop 3.25 support and remove ensure-safe-components again.